hoot-core-test/src/test/cpp/hoot/core/io/OsmApiWriterTest.cpp

```diff
@@ -28,7 +28,6 @@
 //  hoot
 #include <hoot/core/TestUtils.h>
 #include <hoot/core/io/OsmApiWriter.h>
-#include <hoot/core/util/ConfigOptions.h>
```

Comment:
Again all of the changes in this file are mine from another PR and need to be restored.